docs/getting-started/DebuggingGuide.md

```diff
@@ -0,0 +1,50 @@
+# From Zero to Bug Fix Hero
+These steps assume you are an authorized contributor to this repository. If you are not already an authorized contributor, please fork the repo first and complete all steps off of your fork.
+
+## Visual Studio
+Install the latest version of Visual Studio with the minimum initial workloads:
+
+- .NET desktop development
+- Desktop development with C++
```

Comment:
should we link to the Contributing file? In that file, we have a few more things such as the spectre mitigation libraries.